askap/imageaccess/ADIOSImage.tcc

```diff
@@ -95,16 +99,29 @@ void ADIOSImage<T>::makeNewTable(const casacore::TiledShape& shape, casacore::uI
   const casacore::uInt ndim = latShape.nelements();
 
   casacore::TableDesc description;
+  //PJE - why are we adding a hard-coded string???
   description.addColumn(casacore::ArrayColumnDesc<T>("map",
                                                       casacore::String("version 4.0"),
                                                       latShape, casacore::ColumnDesc::FixedShape));
 
   casacore::SetupNewTable newtab(filename, description, casacore::Table::New);
 
-  casacore::Adios2StMan stman("",
+  // now invoke the Adios2 storage manager
+  if (config == "") {
+    // invoke Adios2StMan(engineType, engineParams, transportParams, operatorParams)
+    // with default engine type and parameters 
+    casacore::Adios2StMan stman("",
                               {},
                               {{}},
                               {{{"Variable", "map"},{},{}}});
+  }
+  else 
+  {
+    // invoke configuration based call 
+    //PJE - question on whether we also need to include the operatorParams {{{"Varaible", "map"}}}
```

Comment:
This will need to be in the config file or as an input to the storage manager. I think the variable needs an name...
In the long run, the operatorParams input is where we will put the compression parameters.